plugins/chat/assets/javascripts/discourse/routes/chat-channels.js

```diff
@@ -9,12 +9,6 @@ export default class ChatChannelsRoute extends DiscourseRoute {
     this.chat.activeChannel = null;
   }
 
-  beforeModel() {
```

Comment:
Same point here as mentioned in the routes for direct messages, wondering if we want to keep these redirects?